taipy/gui/.gitignore

```diff
@@ -56,3 +56,5 @@ gui/packaging/taipy-gui.gen.d.ts
 
 # GUI Build
 webapp
+
+!data
```

Comment:
That is too broad a name at the top.
I suspect that for *your* data: *FLLdata*?
Anyway I would not commit that

Reply:
I had to do it as someone committed a change to the root .ignore file with Data on it.
We don't want gui/data to be ignored!